test/test_utils.py

```diff
@@ -669,7 +669,7 @@ def forward(self, x):
             ms(torch.tensor([False], dtype=torch.bool))
 
 
-@unittest.skipIf(IS_SANDCASTLE, "cpp_extension is OSS only.")
+@unittest.skipIf(True, "temporarily disabling")
```

Comment:
Can you make condition a bit tighter?
```suggestion
@unittest.skipIf(IS_SANDCASTLE or IS_WINDOWS, "temporarily disabling")
```

Reply:
yes, done